Makefile

```diff
@@ -1,6 +1,7 @@
 ONYX_DIR=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 USER_ID=$(shell id -u)
 GROUP_ID=$(shell id -g)
+COMPOSER_ARGS=--ignore-platform-reqs
```

Comment:
That's right but does not love this configuration entry. We can not assume the current version of php used by each project bootstrapped with Onyx. We can only enforce requirements.